Source/monster.cpp

```diff
@@ -4140,7 +4140,7 @@ void GolumAi(int monsterId)
 
 void DeleteMonsterList()
 {
-	for (int i = 0; i < MAX_PLRS; i++) {
+	for (size_t i = 0; i < Players.size(); i++) {

```

Comment:
Should this be kept as MAX_PLRS since it works on Golems? Currently it should match but if we resize when a player drops it might not.
```suggestion
	for (size_t i = 0; i < MAX_PLRS; i++) {
```